nodejs/image.bzl

```diff
@@ -115,23 +148,36 @@ def nodejs_image(
             **kwargs
         )
 
+    nodejs_layers = [
+        # Put the Node binary into its own layers.
+        "@nodejs//:node",
+        "@nodejs//:node_files",
```

Comment:
FYI. The `@nodejs` repo will control which node platform is included.
Its an alias repo to the the host system platform. Either `@nodejs_darwin_amd64`, `@nodejs_linux_amd64` or `@nodejs_windows_amd64`.

Reply:
awesome so this hsould be fine with targetting different platforms then